.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+### Describe the change
+
+Explanation of what this PR does
+
+### Where should the reviewer start?
+
+Recommendations for how to test this PR
+
+### Covered unit test cases / E2E test cases
+
+If applicable, explain which unit or E2E tests have been created to test this PR
```

Comment:
While including tests should be a requirement and it's the PR author's responsibility to include tests and it's the PR reviewer's responsibility to ensure tests are included, I'm not sure we should make folks write out what/where the tests are. Maybe a general PR checklist includes relevant tests. Something like:
- [ ] This PR includes tests (unit or E2E).

Reply:
I have added PR checklist section according to your suggestion. What any other item do you think it could be added to PR checklist? In my case I always forget to test in Firefox and the browser checklist would help me to remember. For frontend we have eslint rules so no need to add checklist item, but maybe in backend it would be good to add some `Review types` or similar action.

Reply:
Is it really worth asking people to test on multiple browsers?  I know there are occasional issues, like FF scrollbar in logs, but in general browser incompatibility is rare.

Reply:
Since browser incompatibility is just related to CSS styles, I think a visual quick lookup in the other browser supported does not take too much time.